### PR TITLE
add new press release for micro:bit

### PR DESCRIPTION
**Pull request by jamesadevine**

### CODE REVIEW

File: docs/blog.md [hunk 1]
```diff
@@ -1,5 +1,13 @@
 # Microsoft MakeCode Blog
 
+## [micro:bit shield for MakeCode Arcade](/blog/microbit/arcade-shield)
+
+October 16th, 2020 by [James Devine](https://github.com/jamesadevine)
+
+We are excited to announce that MakeCode Arcade is coming to the new micro:bit via a brand new accessory shield.
```
Comment (pelikhan) [inline docs/blog.md:7]: this sentence reads kind of weird.

File: docs/blog/microbit/arcade-shield.md [hunk 2]
```diff
@@ -0,0 +1,13 @@
+# micro:bit shield for MakeCode Arcade
+
+**Posted on October 16, 2020 by [James Devine](https://github.com/jamesadevine)**
+
+It has been an exciting week with the announcement of the [new micro:bit](https://microbit.org/new-microbit/)! We can’t wait for these new little devices to get into the hands of educators and students next month – having a speaker and microphone will _amplify_ the power of what you can do with these mini-computers, and we can’t wait to see all the creative coding projects you come up with!
+
+As well as the speaker and microphone, the new micro:bit also includes an updated, more powerful processor. What this means is that it will be compatible with [Microsoft MakeCode Arcade](https://arcade.makecode.com/)! MakeCode Arcade is a retro arcade game development environment which makes it super easy for anyone to make a game, blending artistic creativity and computer science education.
```
Comment (pelikhan) [inline docs/blog/microbit/arcade-shield.md:7]: I think we can just say "MakeCode Arcade" here

**Review by pelikhan — CHANGES_REQUESTED** [review]
Reduce .gif or replace with a video/youtube.

**Comment by pelikhan** [comment]
gif is toooooo big.

**Review by Jaqster — APPROVED** [review]
LGTM!

**Pull request merged**